samples/dnn/person_reid.cpp

```diff
@@ -107,10 +107,11 @@ static void extractFeatures(vector<Mat> &imglist, Net &net, vector<Mat> &feature
         Mat out=net.forward();
         vector<int> s {out.size[0], out.size[1]};
         out = out.reshape(1, s);
-        features.resize(out.rows);
         for (int i = 0; i < out.rows; i++)
         {
-            normalize(out.row(i), features[i], 1.0, 0.0, NORM_L2);
+            Mat norm_features;
+            normalize(out.row(i), norm_features, 1.0, 0.0, NORM_L2);
+            features.push_back(norm_features);
         }
```

Comment:
Do you really need to revert it? The old version does the same, but prevent re-allows in the `features` vector.

Reply:
Got the reason. There is outer loop.